app/upgrade_handler.go

```diff
@@ -6,83 +6,21 @@ import (
-	icacontrollertypes "github.com/cosmos/ibc-go/v3/modules/apps/27-interchain-accounts/controller/types"
-	icahosttypes "github.com/cosmos/ibc-go/v3/modules/apps/27-interchain-accounts/host/types"
-	icatypes "github.com/cosmos/ibc-go/v3/modules/apps/27-interchain-accounts/types"
-
-	shieldtypes "github.com/shentufoundation/shentu/v2/x/shield/types"
 )
 
 const (
-	upgradeName = "v2.6.0"
+	upgradeName = "v2.7.0"
 )
 
 func (app ShentuApp) setUpgradeHandler() {
 	app.UpgradeKeeper.SetUpgradeHandler(
 		upgradeName,
 		func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
-			// don't run Initgenesis since it'll be set with a wrong denom
-			fromVM[crisistypes.ModuleName] = app.mm.Modules[crisistypes.ModuleName].ConsensusVersion()
-			// don't run icamodule's Initgenesis since it'll overwrite the icahost params that be set here
-			// the InitModule will be called later on to set params.
-			// this assumes it's the first time ica module go into fromVM
-			fromVM[icatypes.ModuleName] = app.mm.Modules[icatypes.ModuleName].ConsensusVersion()
-
-			ctx.Logger().Info("Start to run module migrations...")
-			// create ICS27 Controller submodule params, controller module not enabled.
-			controllerParams := icacontrollertypes.Params{}
-
-			// create ICS27 Host submodule params
-			hostParams := icahosttypes.Params{
-				HostEnabled: true,
-				AllowMessages: []string{
-					sdk.MsgTypeURL(&banktypes.MsgSend{}),
-					sdk.MsgTypeURL(&stakingtypes.MsgDelegate{}),
-					sdk.MsgTypeURL(&stakingtypes.MsgBeginRedelegate{}),
-					sdk.MsgTypeURL(&stakingtypes.MsgCreateValidator{}),
-					sdk.MsgTypeURL(&stakingtypes.MsgEditValidator{}),
-					sdk.MsgTypeURL(&distrtypes.MsgWithdrawDelegatorReward{}),
-					sdk.MsgTypeURL(&distrtypes.MsgSetWithdrawAddress{}),
-					sdk.MsgTypeURL(&distrtypes.MsgWithdrawValidatorCommission{}),
-					sdk.MsgTypeURL(&distrtypes.MsgFundCommunityPool{}),
-					sdk.MsgTypeURL(&govtypes.MsgVote{}),
-					sdk.MsgTypeURL(&authz.MsgExec{}),
-					sdk.MsgTypeURL(&authz.MsgGrant{}),
-					sdk.MsgTypeURL(&authz.MsgRevoke{}),
-					sdk.MsgTypeURL(&shieldtypes.MsgPurchaseShield{}),
-					sdk.MsgTypeURL(&shieldtypes.MsgWithdrawRewards{}),
-					sdk.MsgTypeURL(&shieldtypes.MsgWithdrawReimbursement{}),
-					sdk.MsgTypeURL(&shieldtypes.MsgDepositCollateral{}),
-					sdk.MsgTypeURL(&shieldtypes.MsgWithdrawCollateral{}),
-				},
-			}
-
-			// initialize ICS27 module
-			icamodule, correctTypecast := app.ModuleManager().Modules[icatypes.ModuleName].(ica.AppModule)
-			if !correctTypecast {
-				panic("mm.Modules[icatypes.ModuleName] is not of type ica.AppModule")
-			}
-			icamodule.InitModule(ctx, controllerParams, hostParams)
-
-			crisisGenesis := crisistypes.DefaultGenesisState()
-			crisisGenesis.ConstantFee.Denom = app.StakingKeeper.BondDenom(ctx)
-			app.CrisisKeeper.InitGenesis(ctx, crisisGenesis)
-
+			// transfer module consensus version has been bumped to 2
```

Comment:
If this line of comment means to show all modules that are going to do upgrading, we could provide a full list of them. Or it may be better to just remove the line.

Reply:
At present, only transfer is upgraded, and other modules are not upgraded for the time being.